src/api/activiti-rest-api/docs/RelatedProcessTask.md

```diff
@@ -0,0 +1,7 @@
+# RelatedProcessTask
+
+## Properties
+Name | Type | Description | Notes
```

Comment:
should be a valid markdown table with `|` at around the columns

Reply:
Fixed